src/main/java/com/nike/cerberus/operation/core/UpdateStackOperation.java

```diff
@@ -47,16 +42,20 @@
     private final CloudFormationService cloudFormationService;
     private final Ec2UserDataService ec2UserDataService;
     private final EnvironmentMetadata environmentMetadata;
+    private final ConfigStore configStore;
 
     @Inject
     public UpdateStackOperation(final CloudFormationService cloudFormationService,
                                 final Ec2UserDataService ec2UserDataService,
-                                final EnvironmentMetadata environmentMetadata) {
+                                final EnvironmentMetadata environmentMetadata,
+                                ConfigStore configStore) {
```

Comment:
nitpick: final should be consistent.  I don't care we are using final on parameters or not.  I'm fine with either style but it should be consistent, especially in the same class.

Reply:
- [ ] make style match